Automattic/wp-calypso
# Login: Use .jsx suffix for the wp-login component

Since this is a component, we should indicate such in the file name.

## To Test
* `git checkout rename/wp-login-component`
* `make run`
* Visit the [Login page](http://calypso.localhost:3000/login)

It should show the login form.
